src/ViURShopClient.js

```diff
@@ -422,6 +422,18 @@ export class ViURShopClient {
             .then(req => req.json());
     }
 
+    discount_remove({
+                        discount_key,
+                    } = {}) {
+        return request(`${this.shop_api_url}/discount_remove`, {
+            method: 'POST',
+            params: this.removeUndefinedValues({
+                discount_key,
+            }),
```

Comment:
The reason for `removeUndefinedValues` was to omit fields that were not provided in the function call (because they are optional).
However, this is superfluous for a single mandatory field.
```suggestion
            params: {
                discount_key,
            },
```